### PR TITLE
driver: Add driver for Attocube ANC300

### PR DESCRIPTION
**Pull request by thibaudruelle**
Changes proposed in this pull request:
- Add a driver for Attocube ANC300 piezo controller with ANMxx0 modules

All the features have been tested with attocube ANC300 (serial:ANC300B-C-1011-1004010, firmware:1.0.0-1271 2012-04-18 16:01), with 6x ANM300 modules in its 6 available slots.

Todo:
- [ ] Test with ANM200 and ANM150 modules, probably make some parameters depend on module model 
- [ ] Implement mock instrument
- [ ] Write tests
- [ ] Write example notebook (WIP)
- [ ] Add more specific exceptions
- [ ] Add software limits
- [ ] Implement output triggers as InstrumentChannel 
- [ ] Improve add_parameter calls (figure out if vals= is necessary when no set_cmd or set_cmd = False)

@WilliamHPNielsen @jenshnielsen

### CODE REVIEW

File: qcodes/instrument_drivers/Attocube/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .devices import VoltageDivider
```
Comment (WilliamHPNielsen) [inline qcodes/instrument_drivers/Attocube/__init__.py]: This relative import is wrong and as far as I can see causing travis to fail. It'd be better to import from `qcodes.instrument_drivers.devices`.

**Comment by thibaudruelle** [comment]
This line got there by mistake, I removed it.

**Comment by WilliamHPNielsen** [comment]
@thibaudruelle what's the status on this work?

**Comment by thibaudruelle** [comment]
@WilliamHPNielsen I've several deadlines to meet in the coming weeks so I unfortunately have not been able to work on this, and won't be in the coming weeks. I'll update this PR as soon as I've done some work on it.

**Comment by WilliamHPNielsen** [comment]
@thibaudruelle, that's completely fine. We'll keep the PR open, then.

**Comment by WilliamHPNielsen** [comment]
@thibaudruelle, as usual, no rush, but what's the status on this PR?

**Comment by astafan8** [comment]
@thibaudruelle, what's the status on this PR?

**Comment by astafan8** [comment]
@thibaudruelle 

The driver in this pull request will not be supported by QCoDeS developers, and, hence, is a **community driver**. Please close this pull-request and open a new one against [`qcodes_contrib_drivers`](https://github.com/QCoDeS/Qcodes_contrib_drivers) repository. QCoDeS developers will close this pull-request unconditionally **in 2 weeks**.

[Read about `qcodes_contrib_drivers` repository and python package here.](https://github.com/QCoDeS/Qcodes_contrib_drivers#qcodes-contrib-drivers)

**Comment by thibaudruelle** [comment]
@astafan8 Thanks for the heads up.

**Pull request closed without merging**